### PR TITLE
[MIG][11.0] stock_split_picking

### PR DESCRIPTION
**Pull request by jcoux**

### CODE REVIEW

File: stock_split_picking/views/stock_partial_picking.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">Stock picking out Split button</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <field name ="state" position="before">
+                <button name="split_process"
+                        states="draft,assigned,partially_available"
+                        string="Split"
+                        confirm="Are you sure you want to split current picking ?"
```
Comment (gurneyalex) [inline stock_split_picking/views/stock_partial_picking.xml]: no space before question mark in english.

**Review by gurneyalex — APPROVED** [review]

File: stock_split_picking/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2015 Camptocamp SA - Nicolas Bessi
+# Copyright 2013-2015 Camptocamp SA - Guewen Baconnier
+# Copyright 2013-2015 Camptocamp SA - Yannick Vaucher
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Split picking',
+    'summary': 'Split a picking in two not transferred pickings',
+    'version': '10.0.1.0.0',
```
Comment (Tonow-c2c) [inline stock_split_picking/__manifest__.py]: 'version': '11.0.1.0.0',

File: stock_split_picking/models/stock_picking.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
+# Copyright 2013-2015 Camptocamp SA - Nicolas Bessi
```
Comment (Tonow-c2c) [inline stock_split_picking/models/stock_picking.py:1]: May be add yours name?

**Review by Tonow-c2c — CHANGES_REQUESTED** [review]

**Review by Tonow-c2c — APPROVED** [review]
LGTM Code review only

File: stock_split_picking/README.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+#. If picking state is **available** you can see an split button.
+#. On the "Operations" tab, fill the field "Done" to the quantity you want to
+   split for each line.
+#. If you click on **Split** button, wizard will split current picking into
+   two different pickings depends on quantity done you entered above.
+#. Both pickings remain confirmed.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/10.0
```
Comment (simahawk) [inline stock_split_picking/README.rst]: /11.0

File: stock_split_picking/README.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (simahawk) [inline stock_split_picking/README.rst]: .png

File: stock_split_picking/README.rst [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+<https://github.com/OCA/stock-logistics-workflow/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```
Comment (simahawk) [inline stock_split_picking/README.rst]: .png

File: stock_split_picking/README.rst [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/stock-logistics-workflow/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
```
Comment (simahawk) [inline stock_split_picking/README.rst:50]: update according to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#credits

File: stock_split_picking/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+{
+    'name': 'Split picking',
+    'summary': 'Split a picking in two not transferred pickings',
+    'version': '11.0.1.0.0',
+    'category': 'Inventory',
+    'author': "Camptocamp, "
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    'license': 'AGPL-3',
+    'website': 'http://www.camptocamp.com',
```
Comment (simahawk) [inline stock_split_picking/__manifest__.py]: https://github.com/OCA/stock-logistics-workflow

File: stock_split_picking/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    'depends': [
+        'stock',
+    ],
+    'data': [
+        'views/stock_partial_picking.xml',
+    ],
+    'installable': True,
```
Comment (simahawk) [inline stock_split_picking/__manifest__.py]: drop this

File: stock_split_picking/models/stock_picking.py [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+            if new_moves:
+                backorder_picking = picking.copy({
+                    'name': '/',
+                    'move_lines': [],
+                    'move_line_ids': [],
+                    'backorder_id': picking.id,
+                })
+                picking.message_post(
+                    _(
+                        'The backorder <a href=# data-oe-model='
```
Comment (simahawk) [inline stock_split_picking/models/stock_picking.py]: is this markup correct? I think it should be:
```html
<a href="#" data-oe-model="stock.picking" data-oe-id="%d">
```

Comment (simahawk) [reply stock_split_picking/models/stock_picking.py]: sounds like somewhere the wrong syntax is used... better to respect HTML anyway.

**Review by simahawk — CHANGES_REQUESTED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**